File: Medical_Profile/blocks.cs
using System.Collections.Generic;

namespace Medical_Profile
{
 public class Ath_block
 {
  public int num = -1;
  public int max_lines = 10;
  public string head = null;
  public string hkey = null;
  public string bkey = null;
  public string[] sortd = null;
  public string endpoint = null;
 }

 public class Blk_info
 {
  public int num = default;
  public int lines = default;
  public int hv = default;
  public int lv = default;
  public int[] ll = null;
 }

 public class Save_blk
 {
  public string Practice;
  public string Patient = null;
  public string Patient_id = null;
  public string Address = null;
  public string DOB = null;
  public string Insurance = null;
  public string Phone = null;
  public string Department;
  public string Sptitle = null;
  public string Secph = null;
  public string Priph = null;
  public string Prtitle = null;
  public string Emergency_contact = null;
  public bool Sec_visible;
  public List<Sblock> Blk_list = new List<Sblock>();
 }

 public class Sblock
 {
  public int num;
  public string header;
  public string body;
 }
}

File: Medical_Profile/blocks.vb

Public Class Ath_block
  Public num As Integer = -1
  Public max_lines As Integer = 10
  Public head As String = Nothing
  Public hkey As String = Nothing
  Public bkey As String = Nothing
  Public sortd() As String = Nothing
  Public endpoint As String = Nothing
End Class

Public Class Blk_info
  Public num As Integer = Nothing
  Public lines As Integer = Nothing
  Public hv As Integer = Nothing
  Public lv As Integer = Nothing
  Public ll() As Integer = Nothing
End Class

Public Class Save_blk
  Public Practice As String
  Public Patient As String = Nothing
  Public Patient_id As String = Nothing
  Public Address As String = Nothing
  Public DOB As String = Nothing
  Public Insurance As String = Nothing
  Public Phone As String = Nothing
  Public Department As String
  Public Sptitle As String = Nothing
  Public Secph As String = Nothing
  Public Priph As String = Nothing
  Public Prtitle As String = Nothing
  Public Emergency_contact As String = Nothing
  Public Sec_visible As Boolean
  Public Blk_list As List(Of Sblock) = New List(Of Sblock)
End Class

Public Class Sblock
  Public num As Integer
  Public header As String
  Public body As String
End Class
